SPECS/fipscheck/fipscheck.spec

```diff
@@ -0,0 +1,156 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Summary:	A library for integrity verification of FIPS validated modules
+Name:		fipscheck
+Version:	1.5.0
+Release:	9%{?dist}
+License:	BSD
+# This is a Red Hat maintained package which is specific to
+# our distribution.
+URL:		https://pagure.io/fipscheck
+Source0:	https://releases.pagure.org/%{name}/%{name}-%{version}.tar.bz2
+
+BuildRequires:	gcc
+BuildRequires:	openssl-devel >= 1.0.0
+
+Requires:      %{name}-lib%{?_isa} = %{version}-%{release}
+
+%description
+FIPSCheck is a library for integrity verification of FIPS validated
+modules. The package also provides helper binaries for creation and
+verification of the HMAC-SHA256 checksum files.
+
+%package lib
+Summary:	Library files for %{name}
+
+Requires:	%{_bindir}/fipscheck
+
+%description lib
+This package contains the FIPSCheck library.
+
+%package devel
+Summary:	Development files for %{name}
+
+Requires:	%{name}-lib%{?_isa} = %{version}-%{release}
+
+%description devel
+This package contains development files for %{name}.
+
+%prep
+%setup -q
+
+%build
+%configure --disable-static
+
+make %{?_smp_mflags}
```

Comment:
Try using `%make_build` and other `make` macros whenever possible. This way we always include Mariner's build flags and configuration.